commit 90b969c5a4dc4381e74a09be875f031b8e95911e
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Thu Feb 9 14:33:30 2017 +0000

    irq-bcm2836: Avoid "Invalid trigger warning"
    
    Initialise the level for each IRQ to avoid a warning from the
    arm arch timer code.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/drivers/irqchip/irq-bcm2836.c b/drivers/irqchip/irq-bcm2836.c
index e8204d9af58f..6e3fab9587b7 100644
--- a/drivers/irqchip/irq-bcm2836.c
+++ b/drivers/irqchip/irq-bcm2836.c
@@ -157,7 +157,7 @@ static void bcm2836_arm_irqchip_register_irq(int hwirq, struct irq_chip *chip)
 
 	irq_set_percpu_devid(irq);
 	irq_set_chip_and_handler(irq, chip, handle_percpu_devid_irq);
-	irq_set_status_flags(irq, IRQ_NOAUTOEN);
+	irq_set_status_flags(irq, IRQ_NOAUTOEN | IRQ_TYPE_LEVEL_LOW);
 }
 
 static void
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1006_90b969c5a4dc4381e74a09be875f031b8e95911e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1006_90b969c5a4dc4381e74a09be875f031b8e95911e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
